src/metrics.go

```diff
@@ -79,10 +79,12 @@ func qCacheHitRatio(metrics map[string]interface{}) (float64, bool) {
 }
 
 func slaveRunningAsNumber(metrics map[string]interface{}) (int, bool) {
-	slaveRunning, ok := metrics["Slave_running"].(string)
-	if ok && slaveRunning == "ON" {
-		return 1, true
-	} else if ok && slaveRunning == "OFF" {
+	slaveIORunning, okIO := metrics["Slave_IO_Running"].(string)
+	slaveSQLRunning, okSQL := metrics["Slave_IO_Running"].(string)
```

Comment:
Seems like these two vars will hold the same metric, is this intended?

Reply:
Not at all. Good catch